quickstarts/ansible-automation-controller/config.yml

```diff
@@ -0,0 +1,45 @@
+
+
+slug: ansible-automation-controller
+description: |
+  ## Why monitor Ansible Automation Controller?
+  An ideal Ansible Automation Controller performance monitor must offer comprehensive and actionable information that software developers need to troubleshoot an application successfully. Some key components are:
+
+  - Track the efficiency of business transactions starting using the Ansible Automation Controller server.
+  - Use deep diagnostics to analyze and fix issues with application performance.
+  - Drill down to the Ansible Automation Controller and the associated application components.
```

Comment:
Why would I do this? What do I get out of "drilling down"? That's what's missing from this bullet. I'm tempted to simply delete it.

Reply:
@paperclypse ,
Deleted the third bullet point. kindly review it.